recipes/python/retail/retail/evaluator.py

```diff
@@ -48,8 +49,17 @@ def split(self, configProperties={}):
                                    user_token=configProperties['ML_FRAMEWORK_IMS_TOKEN'],
                                    service_token=configProperties['ML_FRAMEWORK_IMS_ML_TOKEN'])
 
-        df = prodreader.load(data_set_id=configProperties['trainingDataSetId'],
-                             ims_org=configProperties['ML_FRAMEWORK_IMS_TENANT_ID'])
+        data_set_id = configProperties.get("dataSetId")
```

Comment:
why is there "dataSetId" and "trainingDataSetId"? Shouldn't they be referring to the same dataset?

Reply:
Yes, those are referring to the same dataset. 
The only reason why I left both names is backward compatibility, as there is probably someone who got used to "trainingDataSetId" and now we have to support "dataSetId" (due to service management changes). Also, typically, we will expect dataSetId together with timeframe, as before we did not use it.
If there is no need for backward compatibility, we can keep "dataSetId" and remove "trainingDataSetId"?